pkg/registry/apis/provisioning/lint/dashboard.go

```diff
@@ -8,10 +8,35 @@ import (
 
 	"github.com/grafana/dashboard-linter/lint"
 	provisioning "github.com/grafana/grafana/pkg/apis/provisioning/v0alpha1"
+	"gopkg.in/yaml.v2"
```

Comment:
could/should we depend on v3 or is that from the linter package?

Reply:
the import was added automatically. I will use `v3`.